src/content/en/fundamentals/instant-and-offline/get-started/graphicalcontent-4.md

```diff
@@ -0,0 +1,186 @@
+
+Clearly, one of the major ways to get a significant performance boost is to reduce the amount of time images take to load. Let's examine some ways to tackle this problem.
+
+## Necessity
+
+The first question to ask is whether you really need an image, and it's a question that apparently isn't asked enough. Far too many sites of all kinds put large, and mostly useless, images "above the fold". This slows down the page load and makes readers scroll halfway down the page before they even get to meaningful content. That's bad, bad design.
+
+Yes, images can genuinely support text content or can just lend visual interest, but if an image doesn't add information or clarify meaning for the user, it's not contributing to the reader's experience and you should consider omitting it. The fastest image is one you don't have to download; every image you remove speeds up your page load time.
+
+## Image Size
+
+Of course, large images take longer to download and render than small ones, but the concept of "size" has some nuances.
+
+### Browser-side Resizing
+
+Often, large images are retrieved from the server and then resized in the browser with CSS. This might be for a perfectly logical reason -- such as using just one image for a thumbnail that expands on hover -- or it might just be careless coding. Whatever the reason, it wastes bandwidth.
+
+For example, you might have a 1200 x 600 pixel image that you present at 60 x 30 (as a 5% thumbnail) and roll it up to full size on hover using a CSS transition. It works fine and looks great, but if the user never actually hovers over the thumbnail, then literally 95% of the time that image took to download was wasted. You may be better off taking the time to make separate, properly sized, thumbnails and use them in those locations, displaying the full sized images only if they're actually requested.
+
+Of course, using separate images invokes a tradeoff in additional HTTP requests. It is worth testing to see which technique -- reducing the number of images or reducing the number of HTTP requests -- saves the most load time. You might also consider combining graphical resources as a balanced approach, as described in the next article, **HTTP Requests**.
+
+But thumbnails aren't the only images that need physical resizing. All your images should be appropriately sized for their intended use and should not rely on the browser to resize them for rendering.
+
+### Physical Resizing
+
+Once you've determined the size that an image should really be, how do you get it to that size but still retain the quality and characteristics it needs? The first thing you need is a good image manipulation tool, of which there are many. Here are a few to consider.
+
```

Comment:
XNConvert is a free, cross-platform tool (works on Windows/Mac etc) that can handle batch image resizes, optimization, processing/transforms: https://www.xnview.com/en/xnconvert/

Might be worth including a nod to it.

Reply:
There's also ImageOptim https://imageoptim.com/mac which is heavily used by internal creative teams at Google and has some popularity among Mac users.